plugins/package-configuration-providers/ort-config/src/main/kotlin/OrtConfigPackageConfigurationProvider.kt

```diff
@@ -47,7 +47,7 @@ private const val PACKAGE_CONFIGURATIONS_DIR = "package-configurations"
  */
 @OrtPlugin(
     id = "ORTConfig",
-    displayName = "ort-config",
+    displayName = "ORT Configuration Repository",
```

Comment:
Maybe just "Config" as the repo is also called "ort-config".
Off topic, but I would like to generally move from "configuration" to "config" in the whole code base.